### PR TITLE
use anycloud deploy module

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: avm/src/daemon/daemon.rs [hunk 1]
```diff
@@ -18,7 +17,7 @@ pub async fn get_ip() -> String {
 
 pub async fn start(agz_file: &str, app_id: &str, deploy_token: &str) {
   let app_id = app_id.to_string();
-  let deploy_token = deploy_token.to_string();
+  let _deploy_token = deploy_token.to_string();
```
Comment (cdmistman) [inline avm/src/daemon/daemon.rs:20]: do we even need this?

File: avm/src/daemon/lrh.rs [hunk 2]
```diff
@@ -46,7 +46,7 @@ impl LogRendezvousHash {
 
   // Runs a binary search for the record whose hash is closest to the key hash without
   // going over. If none are found, the *last* record in the list is returned as it wraps around.
-  pub fn get_id_for_key(&self, key: &str) -> &str {
+  pub fn _get_id_for_key(&self, key: &str) -> &str {
```
Comment (cdmistman) [inline avm/src/daemon/lrh.rs:49]: do we even need this?

Comment (depombo) [reply avm/src/daemon/lrh.rs:49]: it will be used later so suppressing the warning for now

File: avm/src/daemon/daemon.rs (hunk 1 above)
Comment (depombo) [reply avm/src/daemon/daemon.rs:20]: it will be used later so suppressing the warning for now

File: avm/Cargo.toml [hunk 3]
```diff
@@ -12,7 +12,10 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-ascii_table = "3.0"
+# New commits to a git dependency are locked in place and only replaced by a `cargo update`.
+anycloud = { git = "https://github.com/alantech/anycloud", branch = "main" }
+# For local development, it is possible to specify a relative path location instead of a git repository
+# anycloud = { path = "../../anycloud/cli" }
```
Comment (dfellis) [inline avm/Cargo.toml:18]: Why does this one have `/cli` at the end but not the git repo above?

Comment (depombo) [reply avm/Cargo.toml:18]: The one above is the github repository and finds the cargo lib recursively, no need (or way) to specify the folder. That one below however specifies a specific folder path.

**Review by cdmistman — APPROVED** [review]

File: avm/Cargo.toml (hunk 3 above)
Comment (dfellis) [reply avm/Cargo.toml:18]: That's bonkers. What if there are two cargo libs in the repo?

**Review by dfellis — APPROVED** [review]
Approved, though still perplexed at the Cargo.toml file.

File: avm/Cargo.toml (hunk 3 above)
Comment (depombo) [reply avm/Cargo.toml:18]: I agree and had the same thought

**Pull request merged**